src/bundler.ts

```diff
@@ -132,7 +132,7 @@ class Bundler {
       return false;
     }
     if (targetBundle.url !== bundle.url) {
-      const relative = this._computeRelativeUrl(bundle.url, targetBundle.url);
+      const relative = urlUtils.relativeUrl(bundle.url, targetBundle.url);
```

Comment:
Oh derp of course it can't because the urls are !== compared by the if.  Derp.  Ignore lol.